python/lsst/sims/photUtils/SedList.py

```diff
@@ -0,0 +1,391 @@
+import os
+import copy
+import numpy
+from lsst.utils import getPackageDir
+from lsst.sims.utils import defaultSpecMap
+from lsst.sims.photUtils import Bandpass, Sed
+
+__all__ = ["SedList"]
+
+class SedList(object):
+    """
+    This class will read in a list of Seds from disk and store them.
+
+    It also has the ability to renormalize, redden (according to the
+    O'Donnell 94, ApJ 422 158 dust model), and redshift the Seds.
+
+    As it reads in the Seds, it will keep track of each unique file it reads
+    in.  If two Seds are based on the same file (before normalization, reddening,
+    etc.), it will refer back to its own memory, rather than reading the
+    file from disk a second time.
+
+    The method loadSedsFromList allows the user to add Seds to the list
+    after the constructor has been called.
+    """
+
+    def __init__(self, sedNameList, magNormList, specMap=defaultSpecMap,
+                 fileDir = getPackageDir('sims_sed_library'),
+                 wavelenMatch = None,
+                 redshiftList = None,
+                 galacticAvList = None,
+                 internalAvList = None,
+                 cosmologicalDimming = True):
+
+        """
+        @param [in] sedNameList is a list of SED file names.
+
+        @param [in] magNormList is a list of magnitude normalizations
+        (in the imsimBandpass) for each of the Seds.
```

Comment:
Yes.  We can add a self._normalizing_bandpass to this class that defaults to the usual imsimbandpass